frontend/src/store/Constants.ts

```diff
@@ -1,4 +1,8 @@
 export const FOREST_CLIENT_INPUT_MAX_LENGTH = 8;
 export const FAM_APPLICATION_NAME = 'FAM';
 export const FAM_APPLICATION_ID = 1;
-export const FAM_ADMIN_ROLE = 'FAM_ADMIN';
\ No newline at end of file
+export const PAGINATOR_TEMPLATE =
```

Comment:
let's rename these three variable to start with "TABLE", like "TABLE_PAGINATOR_TEMPLATE", "TABLE CURRENT_PAGE_REPORT_TEMPLATE", "TABLE_ROWS_PER_PAGE", so we know what it is for in the future, thanks!!

Reply:
just did, thanks!